web/src/test/java/com/epam/rft/atsy/web/controllers/ApplicationStateControllerTest.java

```diff
@@ -271,6 +292,8 @@ public void pageLoadShouldRespondModelAndViewWhenParamApplicationIdAndParamClick
     given(stateFlowService.getStateFlowDTOByFromStateDTO(stateDTOWithStateNameCoding))
         .willReturn(stateFlowDTOListWithSingleElement);
 
+    given(candidateService.getCandidateByApplicationID(APPLICATION_ID)).willReturn(candidateDTO);
```

Comment:
What if the candidateService returns null candidate to the controller?
That is not tested, while the candidate service can return null.